script/openqa-continuous-update

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+set -e
+set -o pipefail
+
+usage() {
+    cat << EOF
+Usage: openqa-continous-update
+Trigger update of packages from devel:openQA if stable
+
+Options:
+ -h, --help         display this help
+EOF
+    exit "$1"
+}
+
+opts=$(getopt -o h --long help -n "$0" -- "$@") || usage 1
+eval set -- "$opts"
+while true; do
+  case "$1" in
+    -h | --help ) usage 0; shift ;;
+    -- ) shift; break ;;
+    * ) break ;;
+  esac
+done
+
+repo_name=${DEVEL_OPENQA_REPOSITORY:-devel_openQA}
+if zypper -n ref -r "$repo_name" | grep -q 'is up to date' && "$(dirname "${BASH_SOURCE[0]}")"/openqa-check-devel-repo; then
```

Comment:
Why do you include `openqa-check-devel-repo` in here as well? IMHO if zypper says it has updated packages, which is from download.opensuse.org or any configured mirror, then we should not care querying the API of OBS. Can we just leave that out?

Reply:
hm, ok. It won't be a promise that the packages that are actually downloadable are ok but I guess that makes sense to be aligned with our current development approach.